stable/prisma/templates/deployment.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "prisma.fullname" . }}
+  labels:
+    app: {{ template "prisma.name" . }}
+    chart: {{ template "prisma.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "prisma.name" . }}
+      release: {{ .Release.Name }}
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: {{ template "prisma.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: prisma
+              containerPort: 4466
+              protocol: TCP
+          env:
+            - name: CONFIG
```

Comment:
I'm not sure Prisma image is using a CONFIG environment variable? Isn't it "PRISMA_CONFIG"?

Reply:
The `CONFIG` environment variable is used by the `prerun_hook.sh` to `envsubst` all the env variables passed.